api/v1alpha1/oadp_types.go

```diff
@@ -160,6 +160,12 @@ type NodeAgentConfig struct {
 	// +kubebuilder:validation:Enum=restic;kopia
 	// +kubebuilder:validation:Required
 	UploaderType string `json:"uploaderType"`
+
```

Comment:
question related to location of field: what will be the behavior of using resticConfig in 1.4? will we allow?

Reply:
I thought about this and came to the conclusion that we want to push users to use nodeagentConfig so we only put support of the server args in nodeagentconfig and not in resticconfig